wagtail/contrib/modeladmin/tests/test_page_modeladmin.py

```diff
@@ -336,81 +336,6 @@ def test_delete_permitted(self):
         )
 
 
-class TestHeaderBreadcrumbs(TestCase, WagtailTestUtils):
```

Comment:
Please add a new test in both test files that the back link exists and goes to the right location.